python/torch_mlir/extras/onnx_importer.py

```diff
@@ -272,6 +274,20 @@ def import_all(self):
         with InsertionPoint(self._b), Location.unknown():
             func_dialect.ReturnOp(outputs)
 
+    def get_none(self):
+        if '' in self._nv_map:
```

Comment:
Just use None as a key. Better to not share the stringy type namespace with onnx

Reply:
Can just change the type of _nv_map to `dict[Optional[str], Value]`

Reply:
> Just use None as a key. Better to not share the stringy type namespace with onnx

So this may seem weird but thats actually on purpose. `onnx` uses empty string as the marker for a missing parameters. By using that as our key we do not need to do anything special for grabbing the `none` value as a parameter.